geomesa-accumulo-bundle/geomesa-accumulo-processors/src/main/resources/META-INF/services/org.apache.nifi.processor.Processor

```diff
@@ -2,3 +2,8 @@ org.geomesa.nifi.processors.accumulo.AvroToPutGeoMesaAccumulo
 org.geomesa.nifi.processors.accumulo.PutGeoMesaAccumulo
 org.geomesa.nifi.processors.accumulo.PutGeoMesaAccumuloRecord
 org.geomesa.nifi.processors.accumulo.UpdateGeoMesaAccumuloRecord
+org.geomesa.nifi.datastore.processor.ConvertToGeoAvro
+org.geomesa.nifi.datastore.processor.geotools.AvroToPutGeoTools
+org.geomesa.nifi.datastore.processor.geotools.PutGeoTools
+org.geomesa.nifi.datastore.processor.geotools.PutGeoToolsRecord
+org.geomesa.nifi.datastore.processor.geotools.UpdateGeoToolsRecord
```

Comment:
Are these going into all the Processors?

Reply:
yes, this is to keep things the same as before. previously they were defined in a META-INF file in geomesa-datastore-processors, but I removed that file so that they wouldn't be in the new modules.